staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager_test.go

```diff
@@ -173,3 +175,86 @@ func TestApplyDoesNotStripLabels(t *testing.T) {
 		t.Fatalf("labels shouldn't get stripped on apply: %v", m)
 	}
 }
+
+func BenchmarkApplyNewObject(b *testing.B) {
+	f := NewTestFieldManager()
+
+	obj := &corev1.Pod{}
```

Comment:
Would it be better to make a new object each time? I don't think we would hit any caching, but maybe better to stay safe?

Reply:
Yes, and this is free anyway so ...

Reply:
Probably doesn't matter, and it doesn't seem feasible for the other tests, so ignore this :)